---
string[] objectName
float64[] confident
float64[] objectPositionXYZ


